core/src/components/datetime/datetime.tsx

```diff
@@ -963,9 +974,9 @@ export class Datetime implements ComponentInterface {
   }
 
   componentWillLoad() {
-    this.processValue(this.value);
     this.processMinParts();
     this.processMaxParts();
+    this.processValue(this.value);
```

Comment:
Based on the current changes, it doesn't need to be, but I would recommend it stays in this position. Originally I had logic that manipulated the defaulted date value for this portion:
```
const valueToProcess = value || getToday();
```

that would take in consideration the min & max parts (which need to be set before this method was called). That work was ultimately reverted, since we need to investigate and discuss how we wish to handle dates outside bounds and if we wish to keep setting the default day (#24456).

Reply:
Sounds good, let's keep it the way you have it.